src/app/connectionStrings/components/connectionStringsView.tsx

```diff
@@ -82,7 +84,7 @@ export const ConnectionStringsView: React.FC = () => {
     React.useEffect(() => {
         if (synchronizationStatus === SynchronizationStatus.upserted) { // only when connection string updated successfully would navigate to device list view
             const hostName = getConnectionInfoFromConnectionString(connectionStringsWithExpiry[0] && connectionStringsWithExpiry[0].connectionString).hostName;
-            history.push(`/${ROUTE_PARTS.RESOURCE}/${hostName}/${ROUTE_PARTS.DEVICES}`);
+            history.push(`/${ROUTE_PARTS.IOT_HUB}/${ROUTE_PARTS.HOST_NAME}/${hostName}/`);
```

Comment:
Believe so -- let me verify.  Basically, we're going a different direction than I thought last summer with the 'azure resource' and so i removed this indirection.

Reply:
ah i think the reason is because of this '<Redirect from={url} to={`${url}/${ROUTE_PARTS.DEVICES}`} />'. so whether having the 'devices' route here results the same